### PR TITLE
[bshoshany-thread-pool] Updated to v3.3.0

### PR DESCRIPTION
**Pull request by bshoshany**
- #### What does your PR fix?
Updated [bshoshany-thread-pool] to v3.3.0

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
all, yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by bshoshany** [comment]
Question: Starting from v3.3.0, the package now has two (stand-alone) header files, `BS_thread_pool.hpp` and `BS_thread_pool_light.hpp`. However, when I tried the port with `*.hpp` in line 9, it generated some errors and didn't install anything. I didn't write the initial port, so I'm not sure how this is supposed to work; how to install both header files?

File: ports/bshoshany-thread-pool/portfile.cmake [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    SHA512 785d6c4827e39b0128501ff4379ba0ffbf90ef4b612eccd0279027e6fc9ce9d2a463c2b8b93515ca0a1dc86dc4c34d9849bc281940b1f490086d1db49d5521bb
+    REF v3.3.0
+    SHA512 980c8dfe90e04cbc622ee4ae2ce67444069311619f0dc5d7ac3b3a3ea59ead42d4c867e669e8ce9c71cdc2a4bae431402a8f5032ab29fdfc1ca507b0e7102d8f
     HEAD_REF master
 )
 
-file(GLOB HEADER_FILES LIST_DIRECTORIES false "${SOURCE_PATH}/*.hpp")
+file(GLOB HEADER_FILES LIST_DIRECTORIES false "${SOURCE_PATH}/BS_thread_pool.hpp")
 
 file(INSTALL "${HEADER_FILES}" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
```
Comment (LilyWangLL) [inline ports/bshoshany-thread-pool/portfile.cmake:11]:
```suggestion
file(INSTALL ${HEADER_FILES} DESTINATION "${CURRENT_PACKAGES_DIR}/include")
```

File: ports/bshoshany-thread-pool/portfile.cmake [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     REPO bshoshany/thread-pool
-    REF v3.2.0
-    SHA512 785d6c4827e39b0128501ff4379ba0ffbf90ef4b612eccd0279027e6fc9ce9d2a463c2b8b93515ca0a1dc86dc4c34d9849bc281940b1f490086d1db49d5521bb
+    REF v3.3.0
+    SHA512 980c8dfe90e04cbc622ee4ae2ce67444069311619f0dc5d7ac3b3a3ea59ead42d4c867e669e8ce9c71cdc2a4bae431402a8f5032ab29fdfc1ca507b0e7102d8f
     HEAD_REF master
 )
 
-file(GLOB HEADER_FILES LIST_DIRECTORIES false "${SOURCE_PATH}/*.hpp")
+file(GLOB HEADER_FILES LIST_DIRECTORIES false "${SOURCE_PATH}/BS_thread_pool.hpp")
```
Comment (LilyWangLL) [inline ports/bshoshany-thread-pool/portfile.cmake:9]:
```suggestion
file(GLOB HEADER_FILES LIST_DIRECTORIES false "${SOURCE_PATH}/*.hpp")
```

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/bshoshany-thread-pool/portfile.cmake (hunk 1 above)
Comment (LilyWangLL) [reply ports/bshoshany-thread-pool/portfile.cmake:11]: You can try this.

**Comment by bshoshany** [comment]
Thanks @LilyWangLL! :)

**Review by LilyWangLL — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**